server/dash/dashboards/neighborhood.py

```diff
@@ -14,16 +14,20 @@
 from design import DISCRETE_COLORS
 from design import LABELS
 
+NEIGHBORHOOD_COUNCILS_NUMBER=99 #Number of nc in Los Angeles as of 06.2022
```

Comment:
Please add spaces around the '='. Are you using an auto-formatter? It should take care of issues like this.

Please read https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments and https://google.github.io/styleguide/pyguide.html#386-punctuation-spelling-and-grammar re: the comment.

Reply:
I added an auto formatter (black). Thanks!

Reply:
> Please add ".idea/*" to ".gitignore". We don't want local IDE configs to be included in the repo

Hi Nich,
I saw that 6 files that I did not modify willingly were added to the PR. Do you know what those are and how would I got about adding those to gitignore?
Thanks!

Reply:
You can add ".idea/*" to the [gitignore](https://github.com/hackforla/311-data/blob/dev/.gitignore) file.